diff --git a/CMakeLists.txt.old b/CMakeLists.txt
index a993012..34afe49 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,7 +117,7 @@ add_subdirectory(include)
 add_subdirectory(src)

 # pkg-config file
-if (UNIX)
+# if (UNIX)
 configure_file(libssh.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/libssh.pc)
 install(
   FILES
@@ -127,7 +127,7 @@ install(
   COMPONENT
     pkgconfig
 )
-endif (UNIX)
+# endif (UNIX)

 # CMake config files
 include(CMakePackageConfigHelpers)
